prov/efa/src/rdm/rxr_rma.c

```diff
@@ -322,6 +370,10 @@ ssize_t rxr_rma_post_write(struct rxr_ep *ep, struct rxr_op_entry *tx_entry)
 	if (peer->is_local && ep->use_shm_for_tx)
 		return rxr_rma_post_shm_write(ep, tx_entry);
 
+	if (rxr_rma_should_write_using_rdma(ep, tx_entry, peer)) {
+		return rxr_op_entry_post_remote_write(tx_entry);
```

Comment:
Although brackets aren't wrong to keep, to maintain consistency in the code you might want to remove them.